docs-user/guide-getting-started.md

```diff
@@ -0,0 +1,114 @@
+
+You can use the popup from the icon, or similarly the devtools panel, by
+clicking on `Start Recording`, then `Capture`. Then the profiler user interface
+will open to show the captured data.
+
+Note that the profiler user interface is an external web page, however it's
+cached locally the first time you access it. This means you'll need a network
+connection to download it the first time you use it. If you have a network
+connection in later uses and a new version is available, the UI will offer to
+switch to the new version.
+
+It's also important to know that the data stays local to your computer until you
+upload it (see below).
+
+### Using the profiler button
+
+It's possible to click on the button to start the profiler with current
+settings. Then click it again to capture the profile and the open the profiler
+UI.
+
+[A video showing how to use the profiler icon](images/getting-started-use-icon.webm ':include :type=video controls')
+
+### Using keyboard shortcuts
+
+The following keyboard shortcuts are available:
+
+* `Ctrl + Shift + 1`: start the profiler, or stop and discard the data if it's already running.
+* `Ctrl + Shift + 2`: capture the profile from the currently running session. It
+  has no effect if the profiler isn't running.
+
+## Share a profile
+
+![A screenshot highlighting the toolbar's upload profile button.](images/getting-started-upload.png)
+
+One of the most powerful and useful features of the Firefox Profiler is the ability to upload and share profiles. The first step is to hit the *Upload Local Profile* button. You'll be able to exclude some information before uploading. Then the profile will be uploaded to an online storage. This profile can then be shared in online chat, emails, and bug reports. Note that anybody that has the link will be access the uploaded data, as it's not protected otherwise.
+
+The current view and all of the filters applied to the profile will be encoded into the URL. After initially sharing the profile, a *Permalink* button will be added, that can then be used to provide a handy shortened URL to the current view.
+
+[A video showing how to upload a profile and how to get the permalink](images/getting-started-upload-permalink.webm ':include :type=video controls')
+
+?> Profiles can also be saved to file, although the current view into the UI will not be saved. The file can be reloaded through the [profiler.firefox.com](https://profiler.firefox.com) interface by either drag and drop, or the file upload interface.
+
+## Delete an uploaded profile
+
+After uploading a profile, you can delete it from the `Profile info` panel.
+
+![A screenshot showing the delete button in the profile info panel](images/getting-started-delete-profile.png)
+
+You can also go to the [Uploaded Recordings page](/uploaded-recordings/) to list
+all profiles you uploaded and delete them. Note that this list is stored locally
+in your Firefox only.
```

Comment:
Do you think it's worth noting that you can only delete the profiles that you uploaded in that browser? (maybe also mentioning that they are stored in IndexedDB, but it might be too much detail for a getting started guide)